pex/testing.py

```diff
@@ -323,6 +323,8 @@ def bootstrap_python_installer(dest):
       break
   else:
     raise RuntimeError("Helper method could not clone pyenv from git after 3 tries")
+  # Create an empty file indicating the fingerprint of the correct set of test interpreters.
+  open(os.path.join(dest, INTERPRETER_SET_FINGERPRINT), 'w').close()
```

Comment:
We have `touch`:
https://github.com/pantsbuild/pex/blob/a4eac2fcf138f1145a033bbbbcd71497d1242b4c/pex/common.py#L196-L199